### PR TITLE
doc(readme): Updated readme with details on support for device stream…

### PR DESCRIPTION
**Pull request by vinagesh**
…ing feature.

## Checklist
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-iot-sdk-csharp/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- [ ] This pull-request is submitted against the `master` branch.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -34,6 +34,14 @@ Due to security considerations, build logs are not publicly available.
 | Microsoft.Azure.Devices.DigitalTwin.Client            | N/A                                                       | [![NuGet][pnp-device-prerelease]][pnp-device-nuget]           |
 | Microsoft.Azure.Devices.DigitalTwin.Service           | N/A                                                       | [![NuGet][pnp-service-prerelease]][pnp-service-nuget]         |
 
+> Note:    
+The device streaming feature is no longer being shipped as part of our newer preview releases as there is no active development going on for this feature in the service. The feature has not been included in any preview release after [2020-10-14](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/preview_2020-10-14). However, the feature is still available under previews/deviceStreaming branch.
```
Comment (abhipsaMisra) [inline readme.md:38]: "Device streaming preview feature has not been included in our newer preview releases..."
We can also add a link to the doc: https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-device-streams-overview

Might also be worth adding a note that prod application should not take dependency on preview nugets - these can include breaking changes

**Review by abhipsaMisra — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (vinagesh) [reply readme.md:38]: All good points. Let me add them.

File: readme.md [hunk 2]
```diff
@@ -34,6 +34,16 @@ Due to security considerations, build logs are not publicly available.
 | Microsoft.Azure.Devices.DigitalTwin.Client            | N/A                                                       | [![NuGet][pnp-device-prerelease]][pnp-device-nuget]           |
 | Microsoft.Azure.Devices.DigitalTwin.Service           | N/A                                                       | [![NuGet][pnp-service-prerelease]][pnp-service-nuget]         |
 
+> Note:    
+Device streaming feature is not being included in our newer preview releases as there is no active development going on in the service. For more details on the feature, see [here](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-device-streams-overview). It is not recommended to take dependency on preview nugets for production applications as breaking changes can be introduced in preview nugets.  
+` `
```
Comment (drwill-ms) [inline readme.md:39]: This puts in some weird highlights in the newlines.

What about this?

> Note:
> Device streaming feature is not being included in our newer preview releases as there is no active development going on in the service. For more details on the feature, see [here](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-device-streams-overview). It is not recommended to take dependency on preview nugets for production applications as breaking changes can be introduced in preview nugets.
>
> The feature has not been included in any preview release after [2020-10-14](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/preview_2020-10-14). However, the feature is still available under previews/deviceStreaming branch.
>
> The latest preview nuget versions that contain the feature are:  
> Microsoft.Azure.Devices.Client - 1.32.0-preview-001
> Microsoft.Azure.Devices - 1.28.0-preview-001

Comment (drwill-ms) [reply readme.md:39]: This works by putting a `>` at the beginning of each line, even the empty ones.

File: readme.md [hunk 3]
```diff
@@ -34,6 +34,16 @@ Due to security considerations, build logs are not publicly available.
...
+> Note:    
+Device streaming feature is not being included in our newer preview releases as there is no active development going on in the service. For more details on the feature, see [here](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-device-streams-overview). It is not recommended to take dependency on preview nugets for production applications as breaking changes can be introduced in preview nugets.  
+` `  
+The feature has not been included in any preview release after [2020-10-14](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/preview_2020-10-14). However, the feature is still available under previews/deviceStreaming branch.   
+` `  
+The latest preview nuget versions that contain the feature are:  
+Microsoft.Azure.Devices.Client - 1.32.0-preview-001  
+Microsoft.Azure.Devices - 1.28.0-preview-001
+
+
```
Comment (drwill-ms) [inline readme.md:46]: remove extra newline here

**Review by drwill-ms — APPROVED** [review]

File: readme.md (hunk 2 above)
Comment (vinagesh) [reply readme.md:39]: On my VS code, that looked just like what you pasted above. However, I was not aware that I can create new lines by just adding '>' in markup. Let me try that out and update.

Comment (vinagesh) [reply readme.md]: '>' works! I udpated the file.

**Pull request merged**